devimint/src/main.rs

```diff
@@ -1040,7 +1040,14 @@ async fn run_ui(
                 Ok(TcpStream::connect(server_addr).await.is_ok())
             })
             .await?;
-            info!(LOG_DEVIMINT, "Started ui/api on http://{server_addr}");
+            let url_protocol = match kind {
+                RunUiKind::Old => "http://",
+                RunUiKind::New => "ws://",
```

Comment:
Don't we want to always use `ws://` or `wss://` ?

Reply:
Good question. I would think `wss` would be the preferred protocol but had not seen it used a lot in the code base so wasn't sure if the API is setup for it. TBH, I wouldn't know where to check if this is the case but will look into it.

Reply:
`wss` only really makes sense in production with a SSL-terminating proxy in front. There should always be two addresses that are configurable:

1. A bind address: That's the one where the server listens on, that's an IP+port and I guess what you are logging here
2. The address the server will be reachable under: that's a protocol+domain/IP+optional port

For this logging we should stick to the unencrypted variant (i.e. `ws://`) since we leave encryption to the deployment (e.g. a NGINX in front).

Reply:
> Don't we want to always use ws:// or wss:// ?

As soon as the old UI is removed, yes, but for now it still starts a HTTP server.